test/common/common_arena_constraints.h

```diff
@@ -38,7 +38,24 @@
 
 #if (_WIN32 || _WIN64) && __TBB_HWLOC_VALID_ENVIRONMENT
 #include <windows.h>
-int get_processors_groups_count() { return GetActiveProcessorGroupCount(); }
+int get_processors_groups_count() {
```

Comment:
To be more consistent with the Windows API:
```suggestion
int get_processor_group_count() {
```